dev.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.49
+
+WORKDIR /app
+
+RUN cargo install systemfd cargo-watch
+
+ENTRYPOINT ["./bin/dev.sh"]
```

Comment:
Would be nice to improve this using multi stage builds and provide some caching to avoid overwhelming CPU with multiple compilation processes on every run